homeassistant/components/sky_remote/__init__.py

```diff
@@ -0,0 +1,58 @@
+"""The Sky Remote Control integration."""
+
+from dataclasses import dataclass
+import logging
+
+from skyboxremote import RemoteControl, SkyBoxConnectionError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+
+PLATFORMS = [Platform.REMOTE]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+type SkyRemoteConfigEntry = ConfigEntry[SkyRemoteData]
+
+
+@dataclass
+class SkyRemoteData:
+    """SkyRemote data type."""
+
+    remote: RemoteControl
```

Comment:
Is there a reason for this type to exist and not just directly passing the RemoteControl object?